Tools/AP_Periph/can.cpp

```diff
@@ -1737,7 +1737,7 @@ void AP_Periph_FW::can_battery_update(void)
         if (battery.lib.get_temperature(temperature, i)) {
             // Battery lib reports temperature in Celsius.
             // Convert Celsius to Kelvin for tranmission on CAN.
```

Comment:
Thank you for changing it to `tranmission` -> `transmission`.

Reply:
Done, thanks